### PR TITLE
Add CHNAGELOG-0.2.0

### PR DESCRIPTION
**Pull request by nannany00**
Add CHANGELOG file for Singleton 0.2.0 release

### CODE REVIEW

File: CHANGELOG-0.2.0.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+#### SDK
+No Update.
+
+
+### Known Issues
+#### Service
+- the Etag in the response header keeps on changing for the same GET request ([#102](https://github.com/vmware/singleton/issues/102))
+- Singleton Service s3 build start failed based on https protocol ([#145](https://github.com/vmware/singleton/issues/145))
+- [Singleton Service][Get pattern by language&region] plurals rule should always follow with language ([#148](https://github.com/vmware/singleton/issues/148))
```
Comment (surite) [inline CHANGELOG-0.2.0.md]: plurals rule does not follow with language

File: CHANGELOG-0.2.0.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- [Singleton Service build] The build failed to start when disable swagger-ui ([#151](https://github.com/vmware/singleton/issues/151))
+- [Singleton Service s3 build] please filter out the file from component list ([#159](https://github.com/vmware/singleton/issues/159))
+- plurals rule should always follow with language ([#169](https://github.com/vmware/singleton/pull/169))
+- [Singleton Service s3 build] Remove extra l10n folder in Singleton Service s3 build ([#194](https://github.com/vmware/singleton/issues/194))
```
Comment (surite) [inline CHANGELOG-0.2.0.md]: As an issue list, the description is not relevant. It looks like a requirement.

File: CHANGELOG-0.2.0.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+
+### Known Issues
+#### Service
+- the Etag in the response header keeps on changing for the same GET request ([#102](https://github.com/vmware/singleton/issues/102))
+- Singleton Service s3 build start failed based on https protocol ([#145](https://github.com/vmware/singleton/issues/145))
+- [Singleton Service][Get pattern by language&region] plurals rule should always follow with language ([#148](https://github.com/vmware/singleton/issues/148))
+- [Singleton Service build] The build failed to start when disable swagger-ui ([#151](https://github.com/vmware/singleton/issues/151))
+- [Singleton Service s3 build] please filter out the file from component list ([#159](https://github.com/vmware/singleton/issues/159))
+- plurals rule should always follow with language ([#169](https://github.com/vmware/singleton/pull/169))
```
Comment (surite) [inline CHANGELOG-0.2.0.md]: The description is same with line 37, but with different issue id, are they different issue?

File: CHANGELOG-0.2.0.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+No Update.
+
+
+### Known Issues
+#### Service
+- the Etag in the response header keeps on changing for the same GET request ([#102](https://github.com/vmware/singleton/issues/102))
+- Singleton Service s3 build start failed based on https protocol ([#145](https://github.com/vmware/singleton/issues/145))
+- [Singleton Service][Get pattern by language&region] plurals rule should always follow with language ([#148](https://github.com/vmware/singleton/issues/148))
+- [Singleton Service build] The build failed to start when disable swagger-ui ([#151](https://github.com/vmware/singleton/issues/151))
+- [Singleton Service s3 build] please filter out the file from component list ([#159](https://github.com/vmware/singleton/issues/159))
```
[the remark this answers is not in the document]
Comment (nannany00) [reply CHANGELOG-0.2.0.md]: It's an issue, if not filter out the file, the API will failed to get translation.

File: CHANGELOG-0.2.0.md (hunk 3 above)
Comment (nannany00) [reply CHANGELOG-0.2.0.md]: removed #169 as it's a PR.

**Pull request merged**